team-2-first-frontend-project/starter-files-folder/starter-files/app-design.html

```diff
@@ -1,58 +1,104 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
   <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicon-32x32.png">
-  
+  <link rel="stylesheet" href="./css/app-design.css">
+
   <title>Frontend Mentor | Designo Agency Website Challenge</title>
 </head>
+
 <body>
-  <ul>
-    <li>Our Company</li>
-    <li>Locations</li>
-    <li>Contact</li>
-  </ul>
-  App design
-  Our mobile designs bring intuitive digital solutions to your customers right at their fingertips.
+  <header>
+    <ul>
+      <li><img src="./assests/" alt="logo" class="logo"></li>
```

Comment:
-  Make another div tag for the logo and keep the logo outside of the  ul tag
- You can also use a nav tag  and inside the nav tag put the ul tag
- The logo image is not displayed. Do check it once.

Reply:
Thanks for the detailed feedback It has been updated and been merged